StardewArchipelago/Locations/CodeInjections/Vanilla/ShippingInjections.cs

```diff
@@ -103,10 +103,33 @@ private void CheckAllShipsanityLocations(List<Item> allShippedItems)
                     _locationChecker.AddCheckedLocation(apLocation);
                 }
                 else
-                {
+                {    
+                    DoBugsCleanup(shippedItem, out var wasSuccessful);
+                    if (wasSuccessful)
+                    {
+                        continue;
+                    }
                     _monitor.Log($"Unrecognized Shipsanity Location: {name} [{shippedItem.ParentSheetIndex}]", LogLevel.Error);
                 }
             }
         }
+
+        private void DoBugsCleanup(Item shippedItem, out bool wasSuccessful)

```

Comment:
BugsCleanup is good to catch it as a "to be removed" thing, but you could add a few words before or after it that describe what it does a little bit as well